### PR TITLE
Autocomplete API change

### PR DESCRIPTION
**Pull request by ibarrick**
search text should now be tracked and initializeable. Typing after something is already selected clears selection.

### CODE REVIEW

**Comment by ibarrick** [comment]
This should be ready now.

File: app/templates/components/paper-autocomplete.hbs [hunk 1]
```diff
@@ -40,6 +40,7 @@
       onKeydown=(action "onKeydown")
       searchEnabled=(readonly searchEnabled)
       searchField=(readonly searchField)
+      searchText=searchText
```
Comment (xomaczar) [inline app/templates/components/paper-autocomplete.hbs]: Can we be explicit here and show that's searchText is one way with readonly helper

File: addon/components/paper-autocomplete-trigger.js [hunk 2]
```diff
@@ -66,11 +74,14 @@ export default Component.extend({
     },
 
     handleInputLocal(e) {
+      // If something is already selected when the user types, it should clear selection
+      if (this.get('selected')) {
+        this.get('select').actions.select(null);
```
Comment (xomaczar) [inline addon/components/paper-autocomplete-trigger.js]: Does it auto search or we need to call actions.search

Comment (ibarrick) [inline addon/components/paper-autocomplete-trigger.js]: the onInput fires and calls search automatically.

File: app/templates/components/paper-autocomplete.hbs (hunk 1 above)
Comment (ibarrick) [inline app/templates/components/paper-autocomplete.hbs]: just added commit to fix this, should be good now.

File: addon/components/paper-autocomplete-trigger.js (hunk 2 above)
Comment (xomaczar) [inline addon/components/paper-autocomplete-trigger.js]: Great

**Comment by xomaczar** [comment]
LGTM - let's merge it.
cc @miguelcobain

**Pull request merged**